source/hardware/controllers/pioneer_ddj_400.rst

```diff
@@ -0,0 +1,185 @@
+on the given page number.
+
+Browser section (p. 6)
+~~~~~~~~~~~~~~~~~~~~~~
+
+.. figure:: ../../_static/controllers/pioneer_ddj_400_browser.svg
+   :align: center
+   :width: 50%
+   :figwidth: 100%
+   :alt: Pioneer DDJ-400 (browser section)
+   :figclass: pretty-figures
+
+   Pioneer DDJ-400 (browser section)
+
+========  ==================================================  ==========================================
+No.       Control                                             Function
+========  ==================================================  ==========================================
+1         :hwlabel:`LOAD` buttons                             Load song into deck.
+2         Rotary Selector                                     Press to toggle focus between the library sidebar and associated panels. Turn to move focus up or down.
+========  ==================================================  ==========================================
+
+Deck sections (p. 6)
+~~~~~~~~~~~~~~~~~~~~
+
+.. figure:: ../../_static/controllers/pioneer_ddj_400_deck.svg
+   :align: center
+   :width: 65%
+   :figwidth: 100%
+   :alt: Pioneer DDJ-400 (deck section)
+   :figclass: pretty-figures
+
+   Pioneer DDJ-400 (deck section)
+
+====  =======================================================  ======================================================================
+No.   Control                                                  Function
+====  =======================================================  ======================================================================
+1     :hwlabel:`BEAT SYNC`                                     Tap to sync tempo to the other playing track. Hold to enable sync lock.
+1     :hwlabel:`SHIFT` + :hwlabel:`BEAT SYNC`                  Cycles through tempo ranges: +/-6%, +/-10%, +/-16%, +/-25%
+2     :hwlabel:`CUE/LOOP CALL`:hwlabel:`>`                     Doubles the currently active loop.
+2     :hwlabel:`SHIFT` + :hwlabel:`CUE/LOOP CALL`:hwlabel:`>`  Jumps a phrase forwards
+3     :hwlabel:`CUE/LOOP CALL`:hwlabel:`<`                     Halves the currently active loop.
+3     :hwlabel:`SHIFT` + :hwlabel:`CUE/LOOP CALL`:hwlabel:`<`  Jumps a phrase backwards
+4     :hwlabel:`RELOOP/EXIT`                                   Activates/deactivates currently set loop
+4     :hwlabel:`shift` + :hwlabel:`RELOOP/EXIT`                Activates current loop and stop playback
+5     :hwlabel:`OUT`                                           Sets end of loop to the current playback position.
+                                                               If quantize is enabled, it is set to the closest beat.
+                                                               If held during an active loop, sets the new end point when released.
+5     :hwlabel:`shift` + :hwlabel:`OUT`                        Toggle :hwlabel:`OUT ADJUST` mode on active loop. Adjust the loop's end position using the jog wheel.
```

Comment:
What is "out adjust" mode? I see this is a label on the device but this is not a term used in Mixxx.

Reply:
How about "Hold to adjust loop's end position with the jog wheel"? Is that what this does?

Reply:
> How about "Hold to adjust loop's end position with the jog wheel"? Is that what this does?
Pretty much exactly that, just the feature is toggled by tapping not holding the button.

Will simplify it as you suggest